public/views/content/studentView.html

```diff
@@ -0,0 +1,3 @@
+<a ui-sref=".addEntry">+ Add new entry</a>
```

Comment:
perhaps not necessary, but I do think it is conventional. Makes it clear its a relative reference.